bears/yaml/TravisLintBear.py

```diff
@@ -24,6 +26,29 @@ class TravisLintBear:
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Formatting', 'Syntax'}
     SEE_MORE = 'https://docs.travis-ci.com/user/travis-lint'
+    CHECK_CONNECTION_URL = 'https://travis-ci.org/'
+
+    @classmethod
+    def check_prerequisites(cls):  # pragma: no cover
+        url_status = cls.get_status_of_url(cls.CHECK_CONNECTION_URL)
```

Comment:
do this after the base check! if the base check now fails, you've made an unnecessary query!